openshift_pipelines/op-release-notes.adoc

```diff
@@ -0,0 +1,36 @@
+//OpenShift Pipelines Release Notes
+include::modules/pipelines-document-attributes.adoc[]
+[id="op-release-notes"]
+= {pipelines-title} Release Notes
+:context: op-release-notes
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+{pipelines-title} is a cloud-native CI/CD experience based on the Tekton project which provides:
+
+* Standard Kubernetes-native pipeline definitions (CRDs).
+* Serverless pipelines with no CI server management overhead.
+* Extensibility to build images using any Kubernetes tool, such as S2I, Buildah, JIB, and Kaniko.
+* Portability across any Kubernetes distribution.
+* Powerful CLI for interacting with pipelines.
+* Integrated user experience with Developer perspective of the {product-title} web console.
```

Comment:
Apologies, I missed a "the" in my suggestion the other day.

"with the Developer perspective"